### DISCUSSION TITLE
What Character encoding are you using?

### DISCUSSION THREAD

**Discussion by timmgleason** [ankitpokhrel/jira-cli#664] [Q&A]
I use two shell environments regularly, depending on the host I am working on. Neither one seems to render the output very clearly (as seen in the example screenshots in the README.md). Both are using **LANG=en_US.UTF-8** and one has **LC_CTYPE="en_US.iso88591"** set (that one is even less readable than the first).
I've searched the discussion boards, documentation, and even looked a bit at the source code. Nothing jumps out pointing to an answer.
I even got an error from a 'jira issue list -w' command _"Error: character set not supported"_. 
I included a couple of redacted screenshots of what I am seeing displayed. The first is the one with a different LC_CTYPE set.
![jira_cli_ss_01](https://github.com/ankitpokhrel/jira-cli/assets/11194213/74561bfe-1a8c-4320-853a-8b78b19c13c4)
![jira_cli_ss_02](https://github.com/ankitpokhrel/jira-cli/assets/11194213/66951a14-a994-4ffb-b253-442f556cdb4e)

Cheers

**Comment by ankitpokhrel** [comment]
Hi @timmgleason, didn't `LC_CTYPE=en_US.UTF-8` work?

**Reply by timmgleason** [reply]
That is the value from the second screen shot.
It has that value in the ENV, but different in 'locale'

(5006)echo $LC_CTYPE                                                
en_us.UTF-8
(5007)locale                                                                    
LANG=en_US.UTF-8
LC_CTYPE="en_US.iso88591"
LC_NUMERIC="en_US.iso88591"
LC_TIME="en_US.iso88591"
LC_COLLATE="en_US.iso88591"
LC_MONETARY="en_US.iso88591"
LC_MESSAGES="en_US.iso88591"
LC_PAPER="en_US.iso88591"
LC_NAME="en_US.iso88591"
LC_ADDRESS="en_US.iso88591"
LC_TELEPHONE="en_US.iso88591"
LC_MEASUREMENT="en_US.iso88591"
LC_IDENTIFICATION="en_US.iso88591"
LC_ALL=en_US.iso88591

**Reply by timmgleason** [reply]
And I have tried different terminals as well. Mobaxterm, konsole, xterm, gnome terminal.
All have the same appearance.

**Comment by Pluckerpluck** [comment]
For the record, I had this exact issue. Didn't seem right when I noticed `--plain` printed just fine.

Then I realised my issue was the fact that it was outputting into `less`, and it was **that** which was screwing up the encoding. Using `export LESS="-r"`  forces `less` to use raw control characters rather than trying to be smart. Fixed the problem for me.

Online man page suggests that using `-r` in the environment variable is actually the same as `export LESS="-R"` but that didn't work for me. So it might be related to my version of `less`. There's also a chance `export LESSCHARSET=utf-8` fixes the issue, but I didn't get a chance to test this as it's a problem at my workplace, and I can't post to github from there.

Anyway, sharing this in case anyone else has the issue. Hopefully it's just working in newer versions nowadays, but for us stuck with old applications, here's a fix.
